### PR TITLE
relnote(109): Stable support for document and element scrollend events

### PR DESCRIPTION
**Pull request by bsmth**
Moving out the experimental note as these features are enabled by default in Fx 109 stable release.

__Related issues and pull requests:__
- https://github.com/mdn/content/issues/22814

__Bugzilla bugs:__

- https://bugzilla.mozilla.org/show_bug.cgi?id=1797013
- https://bugzilla.mozilla.org/show_bug.cgi?id=1803435

### CODE REVIEW

File: files/en-us/mozilla/firefox/releases/109/index.md [hunk 1]
```diff
@@ -42,6 +42,9 @@ This article provides information about the changes in Firefox 109 that will aff
 
 ### APIs
 
+- The `scrollend` events are now supported which indicate that the user has completed scrolling in {{domxref("Element")}} and {{domxref("Document")}} objects.
```
Comment (dlrobertson) [inline files/en-us/mozilla/firefox/releases/109/index.md:45]: nit: there should be a comma right?
```suggestion
- The `scrollend` events are now supported, which indicate that the user has completed scrolling in {{domxref("Element")}} and {{domxref("Document")}} objects.
```

**Review by dlrobertson — COMMENTED** [review]
+1 Looks good to me! Great write up... One very minor nit.

File: files/en-us/mozilla/firefox/releases/109/index.md (hunk 1 above)
Comment (bsmth) [reply files/en-us/mozilla/firefox/releases/109/index.md]: Tnx 👍🏻

**Review by dlrobertson — COMMENTED** [review]
+1 Thanks!

**Review by Rumyra — APPROVED** [review]

**Comment by bsmth** [comment]
Thank you @Rumyra

**Pull request merged**